src/bundle/Core/Resources/config/default_settings.yml

```
@@ -2,6 +2,8 @@ parameters:
     # Kernel related params
     webroot_dir: "%kernel.project_dir%/public"
 
+    trusted_header_client_ip_name: ~
```

naming: as I understand this is our setting, not Symfony one (like the one above)? Maybe then:
```suggestion
    ibexa.trusted_header_client_ip_name: ~
```